packages/block-library/src/navigation-menu/editor.scss

```diff
@@ -154,3 +154,17 @@ $color-control-label-height: 20px;
 		}
 	}
 }
+
+// Initial Placeholder
```

Comment:
I feel this is probably something we should add in the placeholder component or not touch.

Reply:
Let me look into that.

Reply:
Updated this so that any buttons used in Placeholder now have some default margins.